src/table.ts

```diff
@@ -1612,6 +1614,9 @@ Please use the format 'prezzy' or '${instance.name}/tables/prezzy'.`);
             mutationErrorsByEntryIndex.set(originalEntriesIndex, errorDetails);
           });
         })
+        .on('metadata', metadata => {
```

Comment:
I don't think we need to save the metadata for now. When we were debugging the customer's issue, we wanted to see the value of server-timing that's published by GFE. In the future we'll also add this as a metric. But for now I think the changes in tables.ts could be reverted.

Reply:
Ok. I guess the original intention was to make it so that the metadata is available on the stream, but doesn't need to reach the user?